charts/devtron/templates/migrator.yaml

```diff
@@ -366,7 +382,11 @@ spec:
         runAsUser: 1000
       containers:
       - name: postgresql-miscellaneous
-        image: quay.io/devtron/postgres:11.9
+        {{- if not $.Values.installer.arch }}
+        image: {{ include "common.image"   (dict "component" $.Values.components.postgres  "global" $.Values.global ) }}
+        {{- else }}
+        image: {{ include "common.image"   (dict "component" $.Values.components.postgres.arm  "global" $.Values.global ) }}
```

Comment:
Here postgres armImage will be used using extraImage, check if this is working

Reply:
Done